### PR TITLE
feat: Verify Credential Endpoint

### LINKED ISSUE #18: feat: Verify Credential Endpoint
Create verify credential endpoint that will validate credential.

### PR DESCRIPTION
**Pull request by sandrask**
Create verify credential endpoint that will validate credential.

Closes #18

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -27,16 +28,20 @@ import (
...
-	credential               = "credential"
+	credentialStore   = "credential"
+	credentialContext = "https://www.w3.org/2018/credentials/v1"
+
+	// endpoints
 	createCredentialEndpoint = "/credential"
+	verifyCredentialEndpoint = "/verify"
 	createProfileEndpoint    = "/profile"
-	credentialContext        = "https://www.w3.org/2018/credentials/v1"
+	getProfileEndpoint       = "/profile/{" + profilePathVariable
```
Comment (troyronda) [inline pkg/restapi/vc/operation/operations.go]: whats the { for?

Comment (sandrask) [reply pkg/restapi/vc/operation/operations.go]: Good question @talwinder50, I just re-arranged constants so endpoints are together.

File: pkg/restapi/vc/operation/operations.go [hunk 2]
```diff
@@ -92,6 +97,31 @@ func (c *Operation) createCredentialHandler(rw http.ResponseWriter, req *http.Re
...
+	if err != nil {
+		c.writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf("failed to read request body: %s", err.Error()))
+
+		return
+	}
+
+	verified := true
+	message := successMsg
+
+	_, _, err = verifiable.NewCredential(body)
```
Comment (troyronda) [inline pkg/restapi/vc/operation/operations.go]: i assume we need a TODO to validate particular contexts of credentials.

Comment (sandrask) [reply pkg/restapi/vc/operation/operations.go]: Next step is to integrate signing and validation. I will add what is required in the context at that point.

File: pkg/restapi/vc/operation/operations.go [hunk 3]
```diff
@@ -27,16 +28,20 @@ import (
...
+
+	// endpoints
 	createCredentialEndpoint = "/credential"
+	verifyCredentialEndpoint = "/verify"
 	createProfileEndpoint    = "/profile"
-	credentialContext        = "https://www.w3.org/2018/credentials/v1"
+	getProfileEndpoint       = "/profile/{" + profilePathVariable
 	profilePathVariable      = "profileID"
 
-	getProfileEndpoint = "/profile/{" + profilePathVariable
```
Comment (fqutishat) [inline pkg/restapi/vc/operation/operations.go]: ?

**Pull request merged**